Formula/pdns.rb

```diff
@@ -61,19 +64,23 @@ def plist; <<-EOS.undent
     <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
     <plist version="1.0">
     <dict>
-      <key>KeepAlive</key>
-      <true/>
       <key>Label</key>
       <string>#{plist_name}</string>
       <key>ProgramArguments</key>
       <array>
-        <string>#{opt_bin}/pdns_server</string>
+        <string>#{sbin}/pdns_server</string>
```

Comment:
I'll make the change, but what's the philosophy behind `opt_sbin` versus `sbin` since they both point back at the same thing via symlinks?  Knowing would be helpful for future formula edits...